resources/views/test-view-1.php

```diff
@@ -0,0 +1,2 @@
+<?php
+echo'<br> This is test view 1';
```

Comment:
Если Вы уж и захотели ~~выебнуться~~ показать себя знающим PHP, то используйте хотя-бы шорт-теги для такого, а вообще даже PHP тут избыточен.
```suggestion
<?= '<br> This is test view 1'; ?>
```